eng/testing/WasmRunnerTemplate.cmd

```diff
@@ -1,11 +1,16 @@
-@echo off
+@REM @echo off
 setlocal enabledelayedexpansion
 
 :: SetCommands defined in eng\testing\tests.wasm.targets
 [[SetCommands]]
 [[SetCommandsEcho]]
 
 set EXECUTION_DIR=%~dp0
+
+if exist "%EXECUTION_DIR%\package-lock.json" (
```

Comment:
Could we do with flag from specific csproj instead of file sniffing ?
Possibly also via <RunScriptCommands> instead of adding it here.